libtransmission/crypto-utils.c

```diff
@@ -89,17 +89,11 @@ int tr_rand_int(int upper_bound)
 {
     TR_ASSERT(upper_bound > 0);
 
-    int noise;
+    unsigned noise;
```

Comment:
In our whole codebase there're only two places where `unsigned` is not followed by a specific type (in this case `int`), and those two places need to be fixed. Let's not add a third one.

Reply:
```suggestion
    unsigned int noise;
```